### PR TITLE
ci-operator/cri-o: add error handling to ignore image deletion failures

### PR DESCRIPTION
**Pull request by sohankunkerkar**

### CODE REVIEW

File: ci-operator/step-registry/cri-o/setup-fedora/test/cri-o-setup-fedora-test-commands.sh [hunk 1]
```diff
@@ -22,7 +22,9 @@ gcloud compute images create "crio-setup-fedora-${currentDate}" --source-image-f
 # Delete images older than 2 weeks
 images=$(gcloud compute images list --project="openshift-node-devel" --filter="family:crio-setup-fedora AND creationTimestamp<$(date -d '2 weeks ago' +%Y-%m-%dT%H:%M:%SZ)" --format="value(name)")
 if [ -n "$images" ]; then
-    echo "$images" | xargs -I '{}' gcloud compute images delete '{}' --project="openshift-node-devel"
+    echo "The following images will be deleted:"
+    echo "$images"
```
Comment (kannon92) [inline ci-operator/step-registry/cri-o/setup-fedora/test/cri-o-setup-fedora-test-commands.sh:26]: Is this echo really needed?

File: ci-operator/step-registry/cri-o/setup/test/cri-o-setup-test-commands.sh [hunk 2]
```diff
@@ -22,7 +22,9 @@ gcloud compute images create "crio-setup-${currentDate}" --source-image-family="
 # Delete images older than 2 weeks
 images=$(gcloud compute images list --project="openshift-node-devel" --filter="family:crio-setup AND creationTimestamp<$(date -d '2 weeks ago' +%Y-%m-%dT%H:%M:%SZ)" --format="value(name)")
 if [ -n "$images" ]; then
-	echo "$images" | xargs -I '{}' gcloud compute images delete '{}' --project="openshift-node-devel"
+    echo "The following images will be deleted:"
+    echo "$images"
```
Comment (kannon92) [inline ci-operator/step-registry/cri-o/setup/test/cri-o-setup-test-commands.sh:26]: Same here.

Comment (sohankunkerkar) [reply ci-operator/step-registry/cri-o/setup/test/cri-o-setup-test-commands.sh:26]: It's good to have that information while debugging.

**Comment by sohankunkerkar** [comment]
/pj-rehearse periodic-ci-cri-o-cri-o-main-periodics-setup-periodic

**Pull request merged**